userspace/libscap/scap_savefile.c

```diff
@@ -503,11 +508,16 @@ int32_t scap_write_proclist_entry_bufs(scap_t *handle, scap_dumper_t *d, struct
                     scap_dump_writev(d, cgroups, cgroupscnt) != cgroupslen ||
 		    scap_dump_write(d, &rootlen, sizeof(uint16_t)) != sizeof(uint16_t) ||
                     scap_dump_write(d, (char *) root, rootlen) != rootlen ||
+			scap_dump_write(d, &(tinfo->pidns_init_start_ts), sizeof(uint64_t)) != sizeof(uint64_t) ||

```

Comment:
minor: some leading whitespace issues here and below.

Reply:
This file seems overall terribly malformatted, perhaps better to fix it in a cleanup PR.